dist/CHECKSUMS

```diff
@@ -0,0 +1,2 @@
+09e2b9e50396f046390ee400a855c9d97aad7634e1b3b8be324381af3d109ffa  argon2.js
```

Comment:
let's remove this file from dist, it should be only in job artifacts, not in the repo or npm.

Reply:
whoops